server/routes/index.js

```diff
@@ -1,6 +1,14 @@
+require("dotenv").config();
 const express = require("express");
 const router = express.Router();
 
+const { loginRouter } = require("./login.router");
+const { signup } = require("./signup.router");
+const { authenticate } = require("../middleware/authenticate.middle");
+
+router.post("/auth/login", authenticate, loginRouter);
```

Comment:
Is there a reason this route is blocked by middleware? If the user's trying to log in, that typically means they aren't already logged in.

Reply:
There are some problems. I have changed it in this Ticket 2 PR. I will also improve this part when I finish the socket issue.